src/Magento/FunctionalTestingFramework/Console/GenerateTestsCommand.php

```diff
@@ -46,6 +51,15 @@ protected function configure()
                 't',
                 InputOption::VALUE_REQUIRED,
                 'A parameter accepting a JSON string used to determine the test configuration'
+            )->addOption(
```

Comment:
This filter needs to be added in `BaseGenerateCommand`, it's useful for any command that generates tests so it makes sense.
This also means you need to implement the creation of the mftfConfig with the value in all commands that have the argument.